docs/quick_start.rst

```diff
@@ -1,152 +1,21 @@
 Quick Start
 ===========
 
-This guide will walk you through the functionality of IPFX starting with the simplest examples
-and build on itself towards increasing complexity of the scope of the analysis and the size of data to be analyzed.
+To run:
 
-Detect Action Potentials
-------------------------
+.. code-block:: bash
 
-To detect action potentials, you must provide three 1D numpy arrays: stimulus current ``i`` in pA,
-response voltage ``v`` in mV, and timestamps ``t`` in seconds.  With these three arrays, you can then do the following:
+    $ cd ipfx/ipfx/bin
+    $ python pipeline_from_nwb.py input_nwb_file outputdir
```

Comment:
the file that should be used is run_pipeline_from_nwb_file.py

Bad: 
```python pipeline_from_nwb.py input_nwb_file outputdir```

Good: 
```python run_pipeline_from_nwb_file.py input_nwb_file outputdir```

Reply:
revised as suggested. thanks.

Reply:
Apologies for the runaround @gnayuy but Sergey had a much better idea in the comment below.  What do you think about changing this (one more time) to:
```python -m ipfx.bin.run_pipeline_from_nwb_file <input_nwb_file> <outputdir>```

Reply:
have changed to that in my last commit, thanks.